### PR TITLE
fix scylla setup of processing default scylla directories

### PR DESCRIPTION
**Pull request by amoskong**
Fixed two problem in scylla_io_setup
- Problem 1: paths of default directories is invalid, introduced by commit 5ec191536e  /CC @bhalevy 
- Problem 2: wrong path join, introduced by commit 31ddb2145a /CC @syuu1228 

scylla_io_setup, scylla_fstrim and scylla_blocktune.py:
- Fixed default scylla directories when they aren't assigned in scylla.yaml

### CODE REVIEW

**Comment by amoskong** [comment]
Hi @xemul 

scylla-setup in AMI works well, but scylla_setup doesn't work if we executed manually.
We have different code path for AMI or non-AMI.

File: dist/common/scripts/scylla_io_setup [hunk 1]
```diff
@@ -91,12 +91,15 @@ if __name__ == "__main__":
             else:
                 conf_dir = etcdir() + "/scylla"
             cfg = yaml.safe_load(open(os.path.join(conf_dir, "scylla.yaml")))
+
+            if 'workdir' not in cfg or not cfg['workdir']:
+                default_path = datadir()
+            else:
+                default_path = cfg['workdir']
```
Comment (xemul) [inline dist/common/scripts/scylla_io_setup]: This can be rewritten in one line.

    default_path = cfg.get('workdir', '') or datadir()

Comment (amoskong) [reply dist/common/scripts/scylla_io_setup]: Thanks, a simple one:

```
default_path = cfg.get('workdir') or datadir()
```

**Comment by xemul** [comment]
Ack

**Comment by penberg** [comment]
Queued to next. @amoskong, are any of these fixes need for 3.2 or is only `master` affected?

**Comment by amoskong** [comment]
It fixed some problems introduced to master recently. That won't affect 3.2

**Pull request merged**